### PR TITLE
Lesson1

### PR DESCRIPTION
**Pull request by saumya665**

### CODE REVIEW

File: addNum.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+const myfunc=(a,b)=>{
+  const c = a+b;
+  return c;
+};
+console.log(myfunc(2,3));
```
Comment (songz) [inline addNum.js]: remove console.log

File: lesson1/addNumbers.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const myfunc=(a,b)=>{
+  const c = a+b;
+  return c;
```
Comment (allopez7) [inline lesson1/addNumbers.js]: No need for c, just return a+b. Also, rename the function to something more descriptive like add, etc.

File: lesson1/addNumbers.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+}
+myfunc(2,3);
```
Comment (allopez7) [inline lesson1/addNumbers.js]: No need to call the function here. That will be done in the test.

File: lesson1/testAdd.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const addNumbers = require('./addNumbers.js');
+
+const test=(a,b,c)=>{
+  if(addNumbers(a,b)==c)
+  {
+    console.log("Your program is working propery");
+    return true;
```
Comment (SahilKalra98) [inline lesson1/testAdd.js]: Don't need the return statement

**Review by allopez7 — APPROVED** [review]

**Review by allopez7 — APPROVED** [review]

**Review by SahilKalra98 — APPROVED** [review]

File: lesson1/addNumbers.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+const addNumbers=(a,b)=>{
+  const c = a+b;
+  return c;
+}
+module.exports = myfunc;
+//newest update
```
Comment (saumya665) [inline lesson1/addNumbers.js]: remove

File: lesson1/addNumbers.js [hunk 6]
```diff
@@ -0,0 +1,6 @@
+const addNumbers=(a,b)=>{
+  const c = a+b;
```
Comment (saumya665) [inline lesson1/addNumbers.js]: ??

File: lesson1/addNumbers.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
...
+  return c;
+}
+module.exports = myfunc;
```
Comment (saumya665) [inline lesson1/addNumbers.js]: ??

File: lesson1/testAdd.js [hunk 8]
```diff
@@ -0,0 +1,12 @@
+const addNumbers = require('./addNumbers.js');
+
+const test=(a,b,c)=>{
```
Comment (saumya665) [inline lesson1/testAdd.js]: ??

File: lesson1/testAdd.js [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+  if(addNumbers(a,b)==c)
```
Comment (saumya665) [inline lesson1/testAdd.js]: }

File: lesson1/addNumbers.js [hunk 10]
```diff
@@ -0,0 +1,5 @@
+const addNumbers=(a,b)=>{
+  const total = a+b;
+  return total;
```
Comment (songz) [inline lesson1/addNumbers.js]: fyi you can just do `return a + b;`

**Review by songz — APPROVED** [review]

**Pull request merged**